src/cache.js

```diff
@@ -1,57 +0,0 @@
-import flatfile from 'flat-file-db';
-
-function stringifyKey(key) {
-  if (typeof key === 'string') {
-    return key;
-  }
-
-  return JSON.stringify(key);
-}
-
-/**
- * Cache instance to store key value pairs.
- * @type {Class}
- * @param {string} cacheFile The name of the file to cache
- */
-export default class Cache {
-  /**
-   * Constructs the Cache.
-   * @param  {string} cacheFile The filename for the cache.
-   */
-  constructor(cacheFile) {
-    this.db = flatfile.sync(cacheFile);
-  }
-
-  /**
-   * Add new entries to the cache
-   * @param {string}   key  The key that shall be cached
-   * @param {Object}   value The value that should be stored in the cache
-   * @param {Function} callback The callback
-   */
-  add(key, value, callback = () => {}) {
-    this.db.put(stringifyKey(key), value, error => {
-      if (error) {
-        throw error;
-      }
```

Comment:
Not sure if throwing an error here is desired. Better return it with the callback and let the app that uses this lib handle the error. Just breaking isn’t nice. The module itself can work even when the caching fails.

I know, this probably was in the old implementation already. Just read about this some time ago.